app/src/openxr/cpp/OpenXRInputMappings.h

```diff
@@ -215,6 +215,31 @@ namespace crow {
             },
     };
 
+    const OpenXRInputMapping PicoNeo3 {
+            "/interaction_profiles/pico/neo3_controller",
+            IS_6DOF,
+            "vr_controller_pico4_left.obj",
+            "vr_controller_pico4_right.obj",
```

Comment:
This looks good as an approximation. But if you feel like doing it, you better import the 3D models from [here](https://github.com/immersive-web/webxr-input-profiles/tree/main/packages/assets/profiles/pico-neo3) into OBJ format, place them in the same directoy and use them instead of the Pico4's.